resources/views/community/components/user/progression-status/console-progression-list-item.blade.php

```diff
@@ -28,11 +28,19 @@
 
 $displayLabel = $label ?? config('systems')[$consoleId]['name_short'];
 $consoleTooltipLabel = $label ?? config('systems')[$consoleId]['name'];
+
+$targetUser = request('user');
+$cellUrls = [
+    'totals' => route('user.completion-progress', [$targetUser, 'filter[system]' => $consoleId]),
+    'unfinished' => route('user.completion-progress', [$targetUser, 'filter[system]' => $consoleId, 'filter[status]' => 'unawarded']),
+    'any-beaten' => route('user.completion-progress', [$targetUser, 'filter[system]' => $consoleId, 'filter[status]' => 'any-beaten']),
+    'any-mastery' => route('user.completion-progress', [$targetUser, 'filter[system]' => $consoleId, 'filter[status]' => 'gte-completed']),
```

Comment:
There seems to be a calculation difference between the list and the page. My user page says I have 45 NES games mastered:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/cc22c32b-6f48-43f8-8044-15498796ebff)

But the list page says there's 46:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/7f3fb5f1-6888-45f7-9595-0782d01f3c6d)

Reply:
A similar thing happens with my GBC lists. I think it's the "Set has been retired" items that are throwing it off.

Reply:
I think there was a caching issue going on. I'm seeing correct numbers now on latest after updating the cache key:

![Screenshot 2023-09-10 at 6 24 57 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/4d3b3c17-40a3-4b07-88dd-36b18996893f)

Reply:
Which cache key? I commented out this code and didn't see any change:
```
        $cachedAwardedValues = Cache::get(CacheKey::buildUserCompletedGamesCacheKey($user));
        if ($cachedAwardedValues) {
            return getLightweightUsersCompletedGamesAndMax($user, $cachedAwardedValues);
        }
```

Reply:
I'm able to repro with another user- working through this now.

Reply:
Confirmed that totals were not being counted up correctly and some awards were not being filtered out properly from the tallies. Fixes made in https://github.com/RetroAchievements/RAWeb/pull/1833/commits/fd4d4841ddf9c273461d4459c5f2f16dcf63732b and https://github.com/RetroAchievements/RAWeb/pull/1833/commits/d129d8ac6de5fb80a3001870714b8cea2326cded.

Reply:
I still only see 45 in my profile page list, and clicking on it gives the NULL datetime error I mentioned above.

Reply:
Thanks for walking through this with me. I am hoping the changes in the latest commit resolve some of the issues you're seeing locally.